lib/bundler/cli.rb

```diff
@@ -199,6 +199,8 @@ def install
       "Force downloading every gem."
     method_option "ruby", :type => :boolean, :banner =>
       "Update ruby specified in Gemfile.lock"
+    method_option "bundler", :type => :string, :lazy_default => "> 0.a", :banner =>
```

Comment:
It'll only ever pin them to a pre-release if that's the running version of bundler (i.e. if they've installed a pre-release). Remember that RubyGems' default is to run rereleases for binaries